src/money-formatter.js

```diff
@@ -2,11 +2,18 @@
 import currencyFormatData from 'currency-format';
 import { formatDigits, isNumeric } from './utils';
 
-const getCurrencyData = (code) =>
+const getCurrencyData = code =>
   currencyFormatData[code.toUpperCase()];
 
-export const formatSimple = (currencyName, amount, fractionSize) =>
-  `${formatDigits(amount, fractionSize)} ${currencyName}`;
+export const formatSimple = (currencyName, amount, fractionSize) => {
+  const result = [];
+  result.push(formatDigits(amount, fractionSize));
+  if (currencyName && currencyName !== '') {
+    result.push(currencyName);
+  }
+
+  return result.join(' ');
```

Comment:
Maybe it's better not to allocate a new array on each format and use simple string concatenation? For example:

```
const value = formatDigits(amount, fractionSize);
const currency = currencyName ? ` ${currencyName}` : '';
return value + currency;
```